pkg/kubelet/prober/worker.go

```diff
@@ -74,12 +74,14 @@ type worker struct {
 	proberResultsSuccessfulMetricLabels metrics.Labels
 	proberResultsFailedMetricLabels     metrics.Labels
 	proberResultsUnknownMetricLabels    metrics.Labels
+	jitter                              float64
 }
 
 // Creates and starts a new probe worker.
 func newWorker(
 	m *manager,
 	probeType probeType,
+	jitter float64,
```

Comment:
I think this needs some validation. Counterintuitively, [`rand.Float64()`](https://golang.org/pkg/math/rand/#Rand.Float64) always returns a value between [0.0,1.0) but the [max value for a float64](https://golang.org/pkg/math/#pkg-constants) is ginormous (1.8e+308). Can we add that check somewhere?

Reply:
I have added some massaging and renamed vars for clarity. Please have a look.